client/app/views/steps/password.coffee

```diff
@@ -22,6 +22,11 @@ module.exports = class PasswordView extends StepView
         @$visibilityButton = @$('[action=password-visibility]')
         @$visibilityIcon = @$('.icon use')
         @$strengthBar = @$('progress')
+        @$errorContainer=@$('.errors')
+        if @error
+            @handleErrorMessage(@error)
```

Comment:
Good point. But actually handleErrorMessage here is the method to render the error message. Maybe we could rename the ```handleErrorMessage``` to ```renderError```?